cf-acceptance-tests/cf-acceptance-tests_spec.rb

```diff
@@ -58,6 +59,16 @@
     ).to eq(0)
   end
 
+  it "has psql available" do
+    expect(
+      command("psql --version").exit_status
+    ).to eq(0)
+  end
+
+  it "can start postgresql" do
```

Comment:
This description is a bit misleading as the test doesn't actually check if it can start postgresql. It just checks if the ctl binary/scripts exists.

Reply:
right I will change the description to "has postgresql startup script"